src/components/Markdown.svelte

```diff
@@ -20,7 +20,7 @@
   // if inline is set, do not wrap the markdown in a paragraph -- useful for short snippets
   export let inline = true;
 
-  let htmlText = text.replace("<", "&lt;");
+  $: htmlText = text.replace("<", "&lt;");
```

Comment:
Should we add a comment here to explain why we needed to replace this string?

Reply:
Yes, this makes sense. I'll add a quick comment before merging.